cdk/domino_cdk/util.py

```diff
@@ -146,3 +153,60 @@ def overlay(alpha: dict, omega: dict) -> dict:
             return {}
         base_dict = check_type(dictionaries[0])
         return base_dict if len(dictionaries) == 1 else overlay(base_dict, cls.deep_merge(*dictionaries[1:]))
+
+    @staticmethod
+    def create_lambda(
+        scope: cdk.Construct,
+        stack_name: str,
+        dirname: str,
+        name: str,
+        environment: Dict[str, str],
+        resources: List[str],
+        actions: List[str],
+    ) -> cdk.Construct:
+        with open(path.join(dirname, "lambda", name + ".py"), encoding="utf-8") as fp:
+            on_event_code_body = fp.read()
+        on_event = lambda_.Function(
+            scope,
+            name + "_on_event",
+            function_name=f"{stack_name}-{name}",
+            runtime=lambda_.Runtime.PYTHON_3_7,
+            handler="index.on_event",
+            code=lambda_.InlineCode(on_event_code_body),
+            environment=environment,
+            timeout=cdk.Duration.seconds(600),  # default is 3 seconds
```

Comment:
Could wait until it comes up, but depending on what other things we use this for we might find ourselves wanting to configure the timeout.

Reply:
I'll add a parameter "timeout" to create_lambda() when we find that we want to configure it.

Now all lambdas complete when they are complete and do not use the timeout.